anchore_engine/analyzers/malware.py

```diff
@@ -198,6 +206,7 @@ def __init__(self, config, squashed_tar_path, tempdir):
         self.enabled = self.config.get(self.name, {}).get('enabled', False) if self.config is not None and self.config.get(self.name) is not None and self.tar_path is not None else False
         self.refresh_enabled = self.config.get(self.name, {}).get('db_update_enabled', True) if self.config is not None and self.config.get(self.name) is not None else True
         self.tempdir = tempdir
+        self.cmdline_args = self.config.get(self.name, {}).get('cmdline_args', [])
```

Comment:
this could be renamed if anyone thinks of a better name! Maybe **clamscan_args**